task_reactredux/task1/src/views/Counter/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles'
+import Button from '@material-ui/core/Button';
+import { Typography, withStyles, Paper } from '@material-ui/core';
```

Comment:
why you are importing `Button` and `{ Typography, withStyles, Paper }` separately?